app/routes/lending/components/LendableObjectList/index.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  Card,
+  LoadingIndicator,
+  Image,
+  Page,
+  filterSidebar,
+  FilterSection,
+  LinkButton,
+} from '@webkom/lego-bricks';
+import { usePreparedEffect } from '@webkom/react-prepare';
+import { FolderOpen } from 'lucide-react';
+import { Helmet } from 'react-helmet-async';
+import { Link } from 'react-router-dom';
+import { fetchAllLendableObjects } from 'app/actions/LendableObjectActions';
+import abakus_icon from 'app/assets/icon-192x192.png';
+import EmptyState from 'app/components/EmptyState';
+import TextInput from 'app/components/Form/TextInput';
+import { selectAllLendableObjects } from 'app/reducers/lendableObjects';
+import { useAppDispatch, useAppSelector } from 'app/store/hooks';
+import useQuery from 'app/utils/useQuery';
+import styles from './LendableObjectList.module.css';
+import type { ListLendableObject } from 'app/store/models/LendableObject';
+
+const LendableObject = ({
+  lendableObject,
+}: {
+  lendableObject: ListLendableObject;
+}) => {
+  return (
+    <Link to={`/lending/${lendableObject.id}`}>
+      <Card isHoverable hideOverflow className={styles.lendableObjectCard}>
+        <Image
+          className={styles.lendableObjectImage}
+          src={lendableObject.image || abakus_icon}
+          alt={`${lendableObject.title}`}
+        />
+        <div className={styles.lendableObjectFooter}>
+          <h3>{lendableObject.title}</h3>
```

Comment:
I think `h3` is a bit large, and I'm not sure if I like that it's centered. It looks quite different from other list items on the webapp, e.g. events, articles, joblistings etc.